### PR TITLE
fixed package.json link to main

### PR DESCRIPTION
**Pull request by CluEleSsUK**
npm publish does a dry run on sidebranches
also added some integration tests

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,9 +1,15 @@
 {
   "name": "drand-client",
-  "version": "1.0.0-pre.0",
+  "version": "1.0.0-pre.1",
   "description": "A client to the drand randomness beacon network.",
   "type": "module",
-  "main": "lib/drand.ts",
+  "main": "build/index.js",
```
Comment (webmaster128) [inline package.json:6]:
```suggestion
  "main": "build/index.js",
  "types": "build/index.d.ts",
```

I guess you need this too

File: .npmignore [hunk 2]
```diff
@@ -0,0 +1,8 @@
+test
+logo.png
+jest.config.ts
+lib
+tsconfig.json
+.eslintrc.cjs
+.github
+
```
[the remark this answers is not in the document]
Comment (CluEleSsUK) [reply .npmignore:8]: yes true - I added this before adding the `files`; I'll remove it now

File: test/integration.test.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import {fetchBeacon, fetchBeaconByTime, HttpCachingChain, HttpChainClient} from '../lib';
+import 'jest-fetch-mock'
+
+describe('randomness client', () => {
+    const chain = new HttpCachingChain('https://api.drand.sh')
```
Comment (AnomalRoil) [inline test/integration.test.ts:5]: I'd argue tests should be running against the testnet :P

**Review by AnomalRoil — APPROVED** [review]

File: test/integration.test.ts (hunk 3 above)
Comment (CluEleSsUK) [reply test/integration.test.ts]: good point! was testing something against mainnet when I wrote these

**Pull request merged**